packages/itwinui-react/src/core/LabeledSelect/LabeledSelect.tsx

```diff
@@ -108,29 +111,32 @@ export const LabeledSelect = <T,>(
   };
 
   return (
-    <InputContainer
-      label={label}
-      disabled={disabled}
-      required={required}
-      status={status}
-      message={message}
-      icon={displayStyle === 'default' ? icon() : undefined}
-      isLabelInline={displayStyle === 'inline'}
-      className={className}
-      style={style}
-      labelId={labelId}
-    >
+    <InputGrid displayStyle={displayStyle} className={className} style={style}>
+      {label && (
+        <Label required={required} disabled={disabled} id={labelId}>
```

Comment:
should be `as='div'`. check old InputContainer code and make sure #1238 doesn't break

Reply:
Did that, checked that and it still works.